docs/development.md

```diff
@@ -126,9 +165,11 @@ Before running the test suite, both the server and the test environment should b
   * On Windows, use the Task Manager or `taskkill /f /im chromedriver.exe` command.
   * On OS X, use the Activity Monitor or `sudo killall chromedriver` command.
 
-### Running the test suite with Eclipse
+### Running the test suites
+
```

Comment:
We have documentation on how to set ` -Duser.timezone=UTC` on eclipse. Do we need the same for Intellij?